raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -169,8 +169,8 @@ contract TokenNetwork is Utils {
         require(_chain_id > 0);
         require(_settlement_timeout_min > 0);
         require(_settlement_timeout_max > _settlement_timeout_min);
-        require(contractExists(_token_address));
-        require(contractExists(_secret_registry));
+        require(contractExists(_token_address), 'Invalid Token.');
```

Comment:
This uses the same string to the `require(token.totalSupply() > 0)`, to differentiate them I would mention the contract doesnt have code, e.g. 'Token address doesnt have code'

Reply:
And if you change the string for the token address, please also update the one for the secret registry